Field.h

```diff
@@ -125,4 +126,39 @@ class TextField: public Field {
         string getValue() {
             return fieldValue;
         }
-};
\ No newline at end of file
+};
+
+class CharField: public Field {
+    
+    private:
+        string _fieldValue;
+        int _maxLength;
+        
+    public:
+        CharField(string name, int maxLength, string defaultValue="", bool primary=false, bool unique=false, bool nullable=true):
+            Field(name, CHAR, primary, unique, nullable, maxLength) 
```

Comment:
Why are you passing maxLength to the parent class? The last parameter can be left blank in this case.